provider/database_db_homes_resource_test.go

```diff
@@ -0,0 +1,216 @@
+// Copyright (c) 2017, Oracle and/or its affiliates. All rights reserved.
```

Comment:
I'm guessing that this takes a really long time to run. I'd be interested to discuss some improvements offline.

Reply:
should wait on test refactoring task of generator